Source/WebCore/platform/ScrollableArea.cpp

```diff
@@ -68,15 +71,19 @@ ScrollAnimator& ScrollableArea::scrollAnimator() const
     return *m_scrollAnimator.get();
 }
 
-ScrollbarsController& ScrollableArea::scrollbarsController() const
+ScrollbarsController& ScrollableArea::scrollbarsController(Page* page) const
```

Comment:
This is a layering violation; we're down in `/platform/` here and can't know about `Page`.